apps/trading/pages/portfolio/index.page.tsx

```diff
@@ -1,26 +1,76 @@
-      <div className="flex gap-4 mb-12">
-        <AnchorButton href="/portfolio/deposit">{t('Deposit')}</AnchorButton>
-        <AnchorButton href="/portfolio/deposit?assetId=8b52d4a3a4b0ffe733cddbc2b67be273816cfeb6ca4c8b339bac03ffba08e4e4">
-          {t('Deposit tEURO')}
-        </AnchorButton>
-      </div>
-      <div className="flex gap-4">
-        <AnchorButton href="/portfolio/withdrawals">
-          {t('View Withdrawals')}
-        </AnchorButton>
-        <AnchorButton href="/portfolio/withdraw">{t('Withdraw')}</AnchorButton>
-        <AnchorButton href="/portfolio/withdraw?assetId=8b52d4a3a4b0ffe733cddbc2b67be273816cfeb6ca4c8b339bac03ffba08e4e4">
-          {t('Withdraw tEURO')}
-        </AnchorButton>
-      </div>
-    </div>
+    <Web3Container
+      render={() => (
+        <div className="h-full text-ui">
+          <main className="relative h-[calc(100%-200px)]">
+            <aside className="absolute px-[8px] py-[16px] w-[300px] mt-[28px] h-[calc(100%-28px)] w-[300px] overflow-auto">
+              <h2 className="text-h4 text-black dark:text-white">
+                {t('Filters')}
+              </h2>
+            </aside>
+            <section>
+              <GridTabs group="portfolio">
+                <GridTab id="positions" name={t('Positions')}>
+                  <div className={tabClassName}>
+                    <h4 className="text-h4 text-black dark:text-white">
+                      {t('Positions')}
+                    </h4>
+                    <PositionsContainer />
+                  </div>
+                </GridTab>
+                <GridTab id="orders" name={t('Orders')}>
+                  <div className={tabClassName}>
+                    <h4 className="text-h4 text-black dark:text-white">
+                      {t('Orders')}
+                    </h4>
+                    <OrderListContainer />
+                  </div>
+                </GridTab>
+                <GridTab id="fills" name={t('Fills')}>
+                  <div className={tabClassName}>
+                    <h4 className="text-h4 text-black dark:text-white">
+                      {t('Fills')}
+                    </h4>
+                  </div>
+                </GridTab>
+                <GridTab id="history" name={t('History')}>
+                  <div className={tabClassName}>
+                    <h4 className="text-h4 text-black dark:text-white">
+                      {t('History')}
+                    </h4>
+                  </div>
+                </GridTab>
+              </GridTabs>
+            </section>
+          </main>
+          <section className="fixed bottom-0 left-0 w-full h-[200px]">
+            <GridTabs group="collaterals">
+              <GridTab id="collateral" name={t('Collateral')}>
+                <AccountsContainer />
+              </GridTab>
+              <GridTab id="deposits" name={t('Deposits')}>
+                <h2>{t('Deposits...')}</h2>
```

Comment:
chore (blocking): Can we add a link to /portfolio/deposit here. Just so there is an easy way to navigate there for now

Reply:
sure, no problem, added one.